ci_framework/roles/hive/tasks/openstack_unclaim_cluster.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# Deletes the clusterclaim resource as provided
+- name: Deleting a cluster claim
+  vars:
+    claim_name: "{{ cifmw_hive_openstack_claim_name }}"
+    namespace: "{{ cifmw_hive_namespace }}"
+  when: not (cifmw_hive_dry_run | default(false)) | bool
```

Comment:
Would it make sense moving this check into `oc_delete.yml`? It would make it generic when ever there is an extension or leverage of `oc_delete.yml`

Reply:
ack